game-starter/docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  game-starter:
+    image: virtuals-game-starter
+    build:
+      context: .
+      dockerfile: Dockerfile
+    platform: linux/amd64
+    environment:
+      - API_KEY=
+      - WEATHER_API_KEY=
+      - OPENAI_API_KEY=
+      - OPENAI_BASE_URL=
+      - botToken=
+    volumes:
+      - /var/run/tappd.sock:/var/run/tappd.sock
+      - virtuals:/app
+    restart: always
+
+volumes:
+    virtuals:
```

nit: add a newline at end of file to comply with [POSIX standard](https://dev.to/documendous/why-you-should-end-your-source-files-with-a-new-line-156g#:~:text=in%20more%20detail%3A-,POSIX%20Compliance,the%20end%20of%20a%20file.)

```suggestion
    virtuals:

```